commands/show-bl-ip.go

```diff
@@ -71,8 +73,8 @@ func printBLSourceIPs(c *cli.Context) error {
 			bson.M{"blacklisted": true},
 			bson.M{"dat.count_src": bson.M{"$gt": 0}},
 		}}
-	limit := 1000
-	data, err := getBlacklistedIPsResultsView(res, sort, limit, match, "src", "dst")
+	limit := c.Int("limit")
+	data, err := getBlacklistedIPsResultsView(res, sort, c.Bool("no-limit"), limit, match, "src", "dst")
```

Comment:
It seems a bit odd to me to the line above use limit := c.Int("limit) then on this line call that again